basics/response-options.md

```diff
@@ -4,5 +4,23 @@ description: Response
 
 # Response options
 
+The response panel allow you to specify the response details of your endpoint:
 
+![New Endpoint](../Images/newendpoint_details_values.png 'New Endpoint')
 
+In this section you can specify the following options:
+
+- HTTP Status Code
+- Delay
+- Body type
+- Body content
+
+In the **HTTP Status Code** input you can specify the HTTP status code of your response, the example above is using a successfull code, **200**.
+
+In the **Delay** input you can specify the delay (in milliseconds) of your response, the example above doesn't use any delay and the value is **0**.
+
+You can choose between **JSON** or **RAW** as body type of the response.
+
+The **JSON** body must be a valid JSON, you can use the body editor to write and validate the JSON. When the JSON is valid you will see the green tick as the image above. The green icon will be hidden if the JSON is not valid.
+
+In a **RAW** body you can put any text you want.
```

Comment:
I'd write a bit more details about the JSON and the RAW values. Like why would you use the RAW input instead of the JSON editor and add an example for the invalid JSON response as well. Also we could elaborate how the JSON validation works: there's an auto-validation if you click out of the editor's box or switch the toggle to RAW. Also if you start typing there's a delay before validation but the editor is auto-validating in every second.